Packs/IdentityRecordedFuture/Layouts/layoutscontainer-Recorded_Future_Playbook_Alert_Identity_Exposure.json

```diff
@@ -0,0 +1,869 @@
+                                    }
+                                },
+                                "buttonClass": "primary",
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "3de932b0-0717-11ef-b715-15598fdfcb5d",
+                                "index": 2,
+                                "listId": "caseinfoid-b1c78230-0713-11ef-b715-15598fdfcb5d",
+                                "name": "Resolved - True Positive",
+                                "scriptId": "IdentityRecordedFuturePlaybookAlerts|||recordedfuture-identity-playbook-alerts-update",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {
+                                    "add_actions_taken": {
+                                        "simple": "",
+                                        "userMarkedRequired": true
+                                    },
+                                    "alert_ids": {
+                                        "complex": {
+                                            "accessor": "alertid",
+                                            "filters": [],
+                                            "root": "incident",
+                                            "transformers": []
+                                        }
+                                    },
+                                    "comment": {
+                                        "simple": "",
+                                        "userMarkedRequired": true
+                                    },
+                                    "remove_actions_taken": {
+                                        "simple": "",
+                                        "userMarkedRequired": false
+                                    }
+                                },
+                                "buttonClass": "primary",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 22,
+                                "id": "4a6e93d0-0c81-11ef-9ce4-35cb652fb7cd",
+                                "index": 3,
+                                "name": "Select Resolving Action",
+                                "scriptId": "IdentityRecordedFuturePlaybookAlerts|||recordedfuture-identity-playbook-alerts-update",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Recorded Future Set Alert Status",
```

Comment:
May I suggest adding more colors to the buttons? 
For example:
1. Resolved - can be Red
2. In Progress - can be yellow
3. Dismised - can be green 

Adding colors can be intuitive for the analyst :) 
![image](https://github.com/demisto/content/assets/88268646/4eb9a75d-77b5-41c4-985a-be2c171b41d1)

Reply:
Completed to use different colors (Resolved - Red, InProgress - Green, Dismissed - Orange, Set Resolving - Grey)... Response actions are all Blue